pkg_defs/Ska.arc5gl/meta.yaml

```diff
@@ -18,7 +18,6 @@ requirements:
   # Packages required to build the package. python and numpy must be
   # listed explicitly if they are required.
   build:
-    - pip <10.0
     - python
     - setuptools
```

Comment:
While you are at it I propose to move all the explicit `setuptools` dependencies.  Yes, they are a dependence, but this is *always* provided by `python` so this is superfluous.  There are a number of other packages also provided by conda `python` without which everything will break, but we don't explicitly specify those.